python/splendor/human.py

```
@@ -0,0 +1,724 @@
+        """Buy face-up card."""
+        if len(parts) < 3:  # noqa: PLR2004
+            return "Usage: 3 <tier> <index>"
+        tier = int(parts[1])
+        idx = int(parts[2])
+        self.result = BuyCard(tier=tier, index=idx)
+        self.exit()
+        return None
+
+    def _cmd_4(self, parts: list[str]) -> str | None:
+        """Buy reserved card."""
+        if len(parts) < 2:  # noqa: PLR2004
```

Ruff (0.14.6)

Unused `noqa` directive (non-enabled: `PLR2004`)

Remove unused `noqa` directive

(RUF100)